src/setup/helpers.ts

```diff
@@ -139,11 +140,12 @@ export function verifyMultiSigs(
       !seen.has(sigs[i].owner) &&
       allowedPubkeys[sigs[i].owner] &&
       allowedPubkeys[sigs[i].owner] >= requiredSecurityLevel &&
-      ethers.verifyMessage(payload_hash, sigs[i].sig) === sigs[i].owner
+      ethers.verifyMessage(payload_hash, sigs[i].sig)
```

Comment:
Doesn't this remove the check that the sig nature is from the owner entirely?
The issue is that this `===` is case sensitive and it should not be. I don't think the removing the check is correct

Reply:
Arham is OOO. I've updated it. Thanks!